buildbot_nix/__init__.py

```diff
@@ -172,6 +172,11 @@ def run(self) -> Generator[Any, object, Any]:
         # if the command passes extract the list of stages
         result = cmd.results()
         if result == util.SUCCESS:
+            log.info(self.observer.getStderr())
```

Comment:
Could you make the eval errors more visible? As a user, I would see this when looking at the build:

![image](https://github.com/nix-community/buildbot-nix/assets/3248/6beea9f3-edb9-4fbe-8159-7c50a5658f5f)

Then click on two toggles to unveil the eval error:

![image](https://github.com/nix-community/buildbot-nix/assets/3248/bbabe984-e795-4169-8ad3-134f1ad13bf7)

(small nit: the whitespace in front of the first line)

Reply:
Whitespace is already fixed, lefr over from a test and me not knowing how `pre` works. But i dont think we can autoexpand, ill look. But to not put it therw would need UI changes

Reply:
What about the 
![image](https://github.com/nix-community/buildbot-nix/assets/3248/1698de61-1ae8-4524-8364-c95630a0e74c)
Could this be turned orange somehow?

Reply:
I will check, but I dont have high hopes

Reply:
Builbots UI is very configurable, if you want to add/replace views, not if you want to modify them...